SocialFilter/sections/whitelist.txt

```diff
@@ -17,6 +17,9 @@ thehungryjpeg.com,softwarespro.net#@#.fb-like
 wyborcza.pl,wyborcza.biz#@#.socialBar
 ! https://github.com/AdguardTeam/AdguardFilters/issues/56631
 bat.com#@#.socialbutton
+! https://github.com/AdguardTeam/AdguardFilters/issues/58445
+@@||platform.twitter.com/js/button.*.js$domain=nonfungerbils.com
```

Comment:
Why these rules are required?

Reply:
to fix twitter button on the home page ("follow @NonFunGerbiIs").
it should be blue =) but now at home the button looks the same with/without AdGuard
<details>
<img src="https://user-images.githubusercontent.com/18536328/86492698-a24dff80-bd77-11ea-9649-25bdf20ddeb6.png">
</details>

Reply:
How it looks on Mac with these rules:
<img src="https://user-images.githubusercontent.com/18536328/86566607-9e3a0180-bf72-11ea-9994-c337f2138f87.png">